connector_easypost/models/product_packaging.py

```diff
@@ -208,13 +226,23 @@ def height(self, record):
     @mapping
     @changed_by('weight', 'weight_uom_id')
     def weight(self, record):
```

Comment:
Hey @lasley,

Open to ideas here - the product.packaging weight is solely the weight of the packaging, not the packages inside of it. I'm using the picking weight but I'm not sure if there are edge cases where this won't work.

Thanks!